cmd/download.go

```diff
@@ -55,15 +89,42 @@ var downloadCmd = &cobra.Command{
 			return fmt.Errorf("Invalid release specified. X.Y.Z format expected: %s", release)
 		}
 
-		if !versionRE.MatchString(hubVersion) {
-			return fmt.Errorf("Invalid hub-version specified. X.Y.Z format expected: %s", hubVersion)
+		// Either hubVersion or mceVersion must be set
+		if (hubVersion != "" && acmVersion != "") || (hubVersion != "" && mceVersion != "") {
+			return fmt.Errorf("--hub-version is deprecated, and conflicts with --mce-version and --acm-version options")
+		}
+
+		if hubVersion != "" {
+			if !versionRE.MatchString(hubVersion) {
+				return fmt.Errorf("Invalid hub-version specified. X.Y.Z format expected: %s", hubVersion)
+			}
+
+			acmVersion, mceVersion = deprecatedHubVersionToAcmMce(hubVersion)
+		}
+
+		if mceVersion == "" {
+			return fmt.Errorf("MCE version must be specified with --mce-version")
+		}
+
+		if !versionRE.MatchString(mceVersion) {
+			return fmt.Errorf("Invalid mce-version specified. X.Y.Z format expected: %s", hubVersion)
+		}
+
+		if duProfile && acmVersion == "" {
```

Comment:
Bump cobra to 1.5.0 then we can just use something like:	
downloadCmd.MarkFlagsRequiredTogether("du-profile", "acm-version")

Reply:
This would break existing usage